app/storage/encrypted_questionnaire_storage.py

```diff
@@ -22,21 +23,18 @@ def add_or_update(self, data, version):
         questionnaire_state = self._find_questionnaire_state()
         if questionnaire_state:
             logger.debug('updating questionnaire data', user_id=self._user_id)
-            questionnaire_state.state = encrypted_data_json
+            questionnaire_state.q_state = encrypted_data_json
```

Comment:
`q_state`? what does that mean?

Reply:
It means state is a reserved keyword in dynamo, short for questionnaire state

Reply:
We generally don't use abbreviations, as they are not always clear. so this is `questionnaire_state.questionnaire_state` which then seems a bit odd?